apm-collector/apm-collector-worker/src/main/java/org/skywalking/apm/collector/worker/segment/entity/SegmentDeserialize.java

```diff
@@ -1,45 +1,54 @@
 package org.skywalking.apm.collector.worker.segment.entity;
 
-import com.google.gson.stream.JsonReader;
-
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
 import java.io.FileReader;
 import java.io.IOException;
-import java.io.StringReader;
 import java.util.ArrayList;
 import java.util.List;
 
 /**
+ * The <code>SegmentDeserialize</code> provides single segment json string deserialize and segment array file
+ * deserialize.
+ *
  * @author pengys5
+ * @since v3.0-2017
```

Comment:
Why has `since`? Even you want to add this, actually shouldn't, we are not in `v3.0-2017`